UN-OCHA/unocha-site
# UNO-762 Demo content

Part 2 of UNO-601

For screenshots and more info UNO-762

Implements https://github.com/UN-OCHA/common-design-site/pull/348 
1. Performance and e2e tests via Github actions
2. `default_content` module and `unocha_demo_content` custom module with pages for testing

## To test installing the Demo content
1. Check out branch
2. Install site from config `drush si --existing-config -y`
3. Enable custom module `drush en unocha_demo_content -y`

Refs: UNO-762
